catalog-info.yaml

```diff
@@ -1052,3 +1052,39 @@ spec:
           access_level: BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
```

Comment:
This is a duplicate of the one already present (in `main`): https://github.com/elastic/beats/blob/3ba900be92cf4c207787f05a08c6d5d5fcc8fcd2/catalog-info.yaml#L1020-L1054 so let's remove it?

Reply:
I was about to comment something similar, how you were able to test the pipeline while the pipeline itself is not yet merged (thus created). 
From the looks of it, you the above pipeline and this, share the same `pipeline_file` so I assume adding a new backstage entity is something we dont want to do.

Reply:
Done